spec/unit/provider/package/pacman_spec.rb

```diff
@@ -10,6 +10,12 @@
   let(:resource) { Puppet::Type.type(:package).new(:name => 'package', :provider => 'pacman') }
   let(:provider) { described_class.new(resource) }
 
+  if Puppet.features.microsoft_windows?
```

Comment:
So it looks like any place that uses `$CHILD_STATUS` like that... can probably be replaced with:

```ruby
Puppet::Util::Execution.expects(:exitstatus).returns 1
```